docs/ref/validators.txt

```diff
@@ -370,3 +370,28 @@ to, or in lieu of custom ``field.clean()`` methods.
     .. versionchanged:: 5.0
 
         The ``offset`` argument was added.
+
+``DomainNameValidator``
```

Comment:
I think let's move the docs to be either under or above `URLValidator` to keep them close together

Reply:
Now moved to above the `URLValidator` section.